src/Pane.tsx

```diff
@@ -46,8 +52,25 @@ export const InnerPane = ({
   bgColor,
   borderWidth,
   borderColor,
-  children
+  children: _children
 }: PropsWithChildren<InnerPaneProps>) => {
+  const children = useMemo(() => {
+    if (!isValidElement(_children) || typeof _children.type === 'string')
```

Comment:
Things break when `_children` is a Fragment. I think we also need something like,
```ts
if (typeof _children.type === 'symbol')
```

Reply:
Why TypeScript doesn't catch this case is a mystery to me.